---
title: "Genetic variation influences pluripotency"
author: "Selcan Aydin"
date: "`r Sys.Date()`"
output: 
  html_document:
    toc: false
    df_print: paged
    code_folding: hide
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(collapse = TRUE)
```

In my postdoctoral research, I explored the influence of genetic variation on the stability of ground state pluripotency using mouse embryonic stem cells (ESC). As part of a multi-investigator collaboration, we have used genetic mapping on chromatin accessibility, transcript, and protein abundance in a panel of ESCs derived from hundreds of genetically diverse mice. We were able to identify a single causal variant that impacts the expression of *Lifr* that further influences the stability of ground-state pluripotency. More specifically, I contributed as a co-author in two and as a first author in one publication. In the first publication, I was part of the validation effort and analyzed the RNA-seq data from strains where the single nucleotide variant was swapped between two genetic backgrounds with variable stability of pluripotency. My results confirmed the genetic mapping results where the allele-swap gene expression changes were concordant with the allele effect predictions obtained from expression QTL (eQTL). In the accompanying publication, I showed that the expression of signaling pathway effectors were being influenced by genetic variants likely leading to variability in differentiation capacity. Finally, we published the first comprehensive genetic characterization of the pluripotent proteome showing extensive variation in protein abundance across the genetically diverse ESCs where I was the first author. Through multi-omics data integration, I identified proteins and transcripts that were influenced by the variation in *Lifr* expression, previously missed because they lacked QTL. My findings highlight the power of multi-omics data integration in revealing the distal impacts of genetic variation. While QTL mapping with individual traits may be limited due to noise introduced by measurement error, data integration can act to consolidate the influence of genetic signals shared across molecular traits and increase detection power.

-   Skelly DA, Czechanski A, Byers C, **Aydin S**, Spruce C, Olivier C, Choi KB, Gatti DM, Raghupathy N, Stanton A, Vincent M, Dion S, Greenstein I, Pankratz M, Porter DK, Martin W, Qi W, Harrill AH, Choi T, Churchill GA, Munger SC, Baker CL, Reinholdt LA. Mapping the effects of genetic variation on chromatin state and gene expression reveals loci that control ground state pluripotency. Cell Stem Cell 2020 Sep 3;27(3):459-469.e8. PubMed PMID: 32795400; DOI: 10.1016/j.stem.2020.07.005; PubMed Central PMCID: PMC7484384.

-   Ortmann D, Brown S, Czechanski A, **Aydin S**, Muraro D, Huang Y, Tomaz RA, Osnato A, Canu G, Wesley BT, Skelly DA, Stegle O, Choi T, Churchill GA, Baker CL, Rugg-Gunn PJ, Munger SC, Reinholdt LG, Vallier L. Na√Øve pluripotent stem cells exhibit phenotypic variability that is driven by genetic variation. Cell Stem Cell 2020 Sep 3;27(3):470-481.e6. PubMed PMID: 32795399; DOI: 10.1016/j.stem.2020.07.019; PubMed Central PMCID: PMC7487768.

-   **Aydin S**, Pham DT, Zhang T, Keele GR, Skelly DA, Paulo JA, Pankratz M, Choi T, Gygi SP, Reinholdt LG, Baker CL, Churchill GA, Munger SC. Genetic dissection of the pluripotent proteome through multi-omics data integration. Cell Genomics 2023 Mar 23;3(4):100283. PubMed PMID: 37082146; DOI: 10.1016/j.xgen.2023.100283; PubMed Central PMCID: PMC10112288.

See also the very nice preview of our ESC work:

-   D'Antonio M, D'Antonio-Chronowska A, Frazer KA. Revealing Instability: Genetic Variation Underlies Variability in mESC Pluripotency. Cell Stem Cell. 2020 Sep 3;27(3):347-349. doi: 10.1016/j.stem.2020.08.012. PMID: 32888420.

More recently, I have applied our systems genetics approach to neural progenitor cells (NPCs) derived from the same ESCs. We identified many eQTL uniquely observed in NPCs overrepresented for genes involved in cell-cell signaling and nervous system development. Notably, mediation analysis suggests that segregating genetic variants influence ESC transcript abundance which then can impacts variation in gene expression in the NPCs. Currently, I am preparing the results from this project for publication and my future work will further investigate the mechanisms and timing of these effects on NPCs.

